docs/develop/basics.md

```diff
@@ -0,0 +1,117 @@
+---
```

Comment:
I am thinking we should think about making this page the index page of /develop? Thoughts?

Reply:
I'm happy to do that if you like!